src/geode/inspector/criterion/adjacency/brep_meshes_adjacency.cpp

```diff
@@ -89,8 +89,9 @@ namespace geode
                 auto wrong_adjacencies =
                     inspector.polyhedron_facets_with_wrong_adjacency();
                 wrong_adjacencies.set_description(
-                    absl::StrCat( "Block with uuid ", block.id().string(),
-                        " polyhedron facets adjacencies issues" ) );
+                    absl::StrCat( "block ", block.name(),
+                        " polyhedron facets adjacencies issues - uuid ",
```

Comment:
Sometimes you used "uuid", sometimes "UUID". Please homogenize occurences of the same wordings